exonum/src/runtime/error.rs

```diff
@@ -429,6 +430,7 @@ where
 /// [`ErrorKind`]: enum.ErrorKind.html
 /// [`CallSite`]: struct.CallSite.html
 #[derive(Clone, Debug, Fail, BinaryValue)]
+#[derive(Serialize, Deserialize)]
```

Comment:
Is this `derive` still necessary? Same with the `derive(Serialize, Deserialize)` for `ErrorKind`.